src/control-plane/cloudfront-s3-portal.ts

```diff
@@ -185,7 +185,7 @@ export class CloudFrontS3Portal extends Construct {
 
       this.distribution = new CloudFrontWebDistribution(this, 'PortalDistribution', {
         enableIpV6: false,
-        priceClass: PriceClass.PRICE_CLASS_ALL,
+        priceClass: PriceClass.PRICE_CLASS_100,
```

Comment:
Did you test to deploy it in China regions?

Reply:
China region only support 'PriceClass_All'